core/templates/dev/head/base.html

```diff
@@ -237,7 +269,7 @@
                     {% endblock %}
                   </div>
                 </div>
-                </div>
+            </div>
```

Comment:
Something seems a bit off with the indentation here. Could you please check? (Is there an extra closing div somewhere, perhaps?)